apps/storybook/src/HeatmapVisDisplay.stories.tsx

```diff
@@ -1,15 +1,15 @@
-import {
-  Annotation,
-  DefaultInteractions,
-  HeatmapVis,
-  mockValues,
-} from '@h5web/lib';
+import { Annotation, DefaultInteractions, HeatmapVis } from '@h5web/lib';
 import { formatTooltipVal } from '@h5web/shared/vis-utils';
 import type { Meta, StoryFn, StoryObj } from '@storybook/react';
+import ndarray from 'ndarray';
 
 import HeatmapVisStoriesMeta from './HeatmapVis.stories';
 
-const dataArray = mockValues.twoD();
+const { dataArray } = HeatmapVisStoriesMeta.args;
+const dataArrayNoSym = ndarray(
+  dataArray.data.map((val, index) => (index % 41 >= 20 ? val * 5 : val)),
+  dataArray.shape,
+);
```

Comment:
Modifying `twoD` array to break the horizontal symmetry, otherwise the `FlipXAxis` story doesn't make much sense.

Reply:
I can definitely add it to the mock values you're right. I'm not sure about using it in the Cypress test, though, as this would require adding a new dataset to the mock demo or replacing the value of one of the exiting ones. I think it's fine to use the uint8 dataset instead, as it is already asymetrical.

Reply:
> I'm not sure about using it in the Cypress test, though, as this would require adding a new dataset to the mock demo or replacing the value of one of the exiting ones

Ah yes, adding a dataset changes all the screenshots since the explorer is open. Oh well, let's go with `uint8` then.